data/experiment-recipe-samples/desktop-95-multifeature.json

```diff
@@ -0,0 +1,72 @@
+{
+  "appId": "firefox-desktop",
+  "appName": "firefox_desktop",
+  "application": "firefox-desktop",
+  "arguments": {},
+  "branches": [
+    {
+      "features": [
+        {
+          "enabled": true,
+          "featureId": "pocketNewtab",
+          "value": {}
+        },
+        {
+          "featureId": "upgradeDialog",
+          "value": {
+            "enabled": false
+          }
+        }
+      ],
+      "ratio": 1,
+      "slug": "control"
+    },
+    {
+      "features": [
+        {
+          "enabled": true,
+          "featureId": "pocketNewtab",
+          "value": {
+            "compactLayout": true,
+            "lastCardMessageEnabled": true,
+            "loadMore": true,
+            "newFooterSection": true
+          }
+        },
+        {
+          "featureId": "upgradeDialog",
+          "value": {
+            "enabled": true
+          }
+        }
+      ],
+      "ratio": 1,
+      "slug": "treatment"
+    }
+  ],
+  "bucketConfig": {
+    "count": 10000,
+    "namespace": "firefox-desktop-multifeature-test",
+    "randomizationUnit": "normandy_id",
+    "start": 0,
+    "total": 10000
+  },
+  "channel": "nightly",
```

Comment:
The 95 Firefox version for desktop is currently the Release type.

Reply:
Are you able to pull down previous builds of nightly at the specific version?

Reply:
We ended up using the Release build with the channel changed to Nightly.